build-scripts/publish.js

```diff
@@ -29,7 +29,7 @@ const path = require('path');
 const runCommand = require('./run-command');
 
 const packagesDirPath = path.resolve(__dirname, '../packages');
```

Comment:
This looks unused to me.

Reply:
`packagesDirPath` is used on Line 89.
The `npmrcPath` was unused (I missed that in the previous PR), but is used now as part of this PR on Line 61.

Reply:
Ah, yes, I missed the expanding there.